cmd/orb.go

```diff
@@ -175,6 +176,23 @@ Example: 'circleci orb publish increment foo/orb.yml foo/bar minor' => foo/bar@1
 	publishCommand.AddCommand(promoteCommand)
 	publishCommand.AddCommand(incrementCommand)
 
+	unlistCmd := &cobra.Command{
+		Use:   "unlist <namespace>/<orb> <true|false>",
+		Short: "Disable or enable an orb's listing in the registry",
+		Long: `Disable or enable an orb's listing in the registry.
```

Comment:
We should include an explicit warning that unlisted orbs are still public and can be consumed if the name is known.